script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -373,7 +373,7 @@ Arihiko slurps his Chikara udon.//　ぞぞぞ、と力うどんをすする有
 He was so absorbed in the conversation with Senpai that it got cold.
 <Page117>
 "You were after Senpai, right?"
```

Comment:
Would "You were after Senpai, weren't you?" be better?

Reply:
agreed